src/hooks/useSignInDialog/Steps/Steps.style.ts

```diff
@@ -1,28 +1,41 @@
 import { Text } from '@/shared/components'
+import { SvgGlyphWarning } from '@/shared/icons/GlyphWarning'
 import { sizes, colors } from '@/shared/theme'
 import styled from '@emotion/styled'
 
 type StepWrapperProps = {
   centered?: boolean
+  withBottomBar?: boolean
 }
 
 export const StepWrapper = styled.div<StepWrapperProps>`
   width: 100%;
   text-align: ${({ centered }) => (centered ? 'center' : 'left')};
   display: flex;
+  flex-grow: 1;
   flex-direction: column;
   align-items: ${({ centered }) => (centered ? 'center' : 'flex-start')};
   margin-top: ${({ centered }) => (centered ? sizes(12) : sizes(2))};
 `
 
 export const StepTitle = styled(Text)`
-  line-height: ${sizes(8)};
-  margin-top: ${sizes(4)};
-  max-width: 400px;
+  margin-top: ${sizes(2)};
 `
 
 export const StepSubTitle = styled(Text)`
   margin-top: ${sizes(2)};
-  color: ${colors.gray[200]};
   max-width: 400px;
 `
+
+export const StepFooter = styled.div`
+  margin-top: 24px;
+  background-color: ${colors.gray[800]};
+  width: 100%;
+  outline: 24px solid ${colors.gray[800]};
```

Comment:
I suggest we don't do that. This may make the layout really hard to understand, it took me some time. How about this instead:
```css
margin: 0 calc(-1 * var(--dialog-padding));
padding: 0 var(--dialog-padding);
```
wdyt?

Reply:
yeah, that's definitely a better solution, but to make this work I also had to change `width` a bit. Fixed